latest/index.bs

```diff
@@ -467,15 +468,16 @@ labeled multiscale image(s). All label images SHOULD be listed within this metad
 ```
 
 The `.zattrs` file for the label image MUST implement the multiscales specification. Within the `multiscales` object, the JSON array 
-associated with the `datasets` key MUST have the same number of entries (scale levels) as the original unlabeled image. 
+associated with the `datasets` key MAY have the same number of elements (scale levels) as the original unlabeled image. 
+Therefore it is acceptable for a label image to have fewer scale levels than the original image.
 
-In addition to the `multiscales` key, the JSON object in this image-level `.zattrs` file SHOULD contain another key, `image-label`, 
-whose value is also a JSON object. The `image-label` object stores information about the display colors, source image, and optionally, 
+The label image .zattrs file MUST also contain the `image-label` key, whose value is a JSON object. 
+The `image-label` object stores information about the display colors, source image, and optionally, 
```

Comment:
Since there are no mandatory contents of the `image-label` dictionary, it would be valid to have `image-label: {}`.
It seems that making the spec more strict (a breaking change) to require the `image-label` dict is probably not worth it in that case?

Reply:
Thanks for the ping @virginiascarlett. Trying to justify, the rationale behind my original statement, I see a label image as a specialized type of multiscales image. At the moment, the specification enforces that such data must be stored within a well-defined `labels` hierarchy but moving forward, I could certainly imagine a relaxation of this constraint.

A typical use case that comes immediately to mind is the one where segmentation / classification is performed against a read-only Zarr dataset e.g. public data and the output of this process needs to be stored as a new dataset. At the moment, the structure which is the most compliant with the spirit of the specification is create an artificial `labels/<label_name>/` hierarchy under the root even if there is no multiscales image. Assuming we relaxed this constraint to allow label images to be stored at the root of the Zarr dataset, I would argue the `image-label` metadata would become a critical element to identify what we are dealing with.

That being said, Will's point makes sense and if this specification change simply leads to most implementations adding `image-label: {}` in the metadata, I don't find this is particularly helpful.
Happy to separate this discussion from your proposal, revert the requirement level back to the SHOULD level and come back to this discussion if we decide to specify standalone label images.